docs/fundamentals/external-parameters.md

```diff
@@ -53,6 +53,31 @@ var value = builder.Configuration[key]; // value = "local-value"
 > [!IMPORTANT]
 > However, you don't need to access this configuration value yourself in the app host. Instead, the <xref:Aspire.Hosting.ApplicationModel.ParameterResource> is used to pass the parameter value to dependent resources. Most often as an environment variable.
 
+### Prompt for parameter values in the dashboard
+
+From .NET Aspire version 9.4 onwards, if your code adds parameters but doesn't set them, then you will see a prompt to configure their values in the .NET Aspire dashboard. The **Unresolved parameters** message is displayed and you can click **Enter values** to resolve the problem:
+
+:::image type="content" source="./media/dashboard-unresolved-parameters-message.png" lightbox="./media/dashboard-unresolved-parameters-message.png" alt-text="Screenshot of the .NET Aspire dashboard warning that appears when there are unresolved parameters.":::
+
+When you select **Enter values**, .NET Aspire displays a form that you can use to configure values for each of the missing parameters.
+
+You can also control how the dashboard displays these parameters, by using these methods:
+
+- `WithDescription`: Use this method to provide a text description that helps users to complete the parameter.
+- `WithMarkdownDescription`: Use this method to provide a formatted description in [Markdown](https://www.markdownguide.org/basic-syntax/) that helps users to complete the parameter.
```

Comment:
I'm not sure I like the phrase "complete the parameter", the description is intended to accompany the input prompt on screen. It's really just helping describe what the input is intended for. I don't think I'd say that a user ever completes a parameter, what do you think about this instead?

```suggestion
- `WithDescription`: Use this method to provide a text description that helps users understand the purpose of the parameter.
- `WithMarkdownDescription`: Use this method to provide a formatted description in [Markdown](https://www.markdownguide.org/basic-syntax/) that explains the parameter's intent.
```